ge/org.osate.ge/src/org/osate/ge/DiagramExporter.java

```diff
@@ -46,13 +48,8 @@
 import org.osgi.framework.FrameworkUtil;
 
 public class DiagramExporter {
-	/**
-	 * Converts an AADL diagram file to a PNG image.
-	 *
-	 * @param  diagramFile the diagram file to be exported
-	 * @param  outputFile  the file the image will be written to
-	 */
-	public static void exportDiagramAsPng(final IFile diagramFile, final File outputFile) throws IOException {
+
+	public static org.eclipse.graphiti.mm.pictograms.Diagram readDiagram(final IFile diagramFile) throws IOException {
```

Comment:
Is there a need for this to  be private? Since diagram exporter is part of the GE's API, I would prefer not to expose this method unless there is a compelling reason.

Reply:
No, there is no compelling reason for that method to be public.